src/index.html

```diff
@@ -17,11 +17,379 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="/src/styles/index.scss"
```

Comment:
The href attribute is pointing to a SCSS file. Browsers do not understand SCSS directly; it needs to be compiled to CSS first. Ensure that the SCSS is compiled to a CSS file and update the href to point to the resulting CSS file.